package.json

```diff
@@ -33,7 +33,8 @@
     ]
   },
   "eslintIgnore": [
-    "coverage/**"
+	"coverage/**",
+	"packages/*/node_modules/**"
```

Comment:
Why this?

Reply:
When installing two versions of dependencies in 2 different packages inside `ckeditor5-dev`'s subrepos, both aren't moved to the `ckeditor5-dev` by Lerna.